### PR TITLE
IR docs v2

### PR DESCRIPTION
**Pull request by justinegeffen**
Second iteration of the docs.

### CODE REVIEW

File: docs/administrator-guide/incident-response-slash-commands.md [hunk 1]
```diff
@@ -1 +1,31 @@
-# Creating and Using Slash Commands
+# Slash Commands
+
+Slash commands are shortcuts used to perform actions in Mattermost. To view the available slash commands in Mattermost begin by typing / and a list of slash command options appears above the text input box. The autocomplete suggestions help by providing a format example in black text and a short description of the slash command in grey text.
```
Comment (justinegeffen) [inline docs/administrator-guide/incident-response-slash-commands.md:3]:
```suggestion
Slash commands are shortcuts used to perform actions in Mattermost. To view the available slash commands in Mattermost begin by typing `/` and a list of slash command options appears above the text input box. The autocomplete suggestions help by providing a format example in black text and a short description of the slash command in grey text.
```

File: docs/administrator-guide/overview.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # Overview
 
-Incident Response is a Mattermost plugin designed to help organizations monitor, coordinate, and measure their incident response processes, increasing transparency, maximizing effectiveness, and ultimately saving costs by cutting down time taken to respond and resolve incidents.
+Incidents are situations which require an immediate response and benefit from a clearly defined process guiding towards resolution. Incident Response allows your team to coordinate, manage, and resolve incidents from within Mattermost.
 
-Incidents are situations which require an immediate response and benefit from a clearly defined process guiding towards resolution. A playbook defines this process, initializing a dedicated channel for the incident with steps grouped into stages for members of the incident to follow. When the situation is resolved, the incident is ended, and the playbook can be updated to improve the response to similar incidents in the future.
+![Incident Response plugin screenshot](assets/center_channel.png)
```
Comment (amyblais) [inline docs/administrator-guide/overview.md:5]: Image is broken

File: docs/administrator-guide/overview.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
...
 
-Playbooks and incidents are scoped by teams, and cannot be shared between teams.
+When incidents are monitored, coordinated, and measured effectively you can transparency, maximize effectiveness, and save costs by cutting down time taken to respond and resolve incidents.
```
Comment (amyblais) [inline docs/administrator-guide/overview.md:7]:
```suggestion
When incidents are monitored, coordinated, and measured effectively, you can add transparency, maximize effectiveness, and save costs by cutting down time taken to respond and resolve incidents.
```

File: docs/administrator-guide/overview.md (hunk 2 above)
Comment (justinegeffen) [reply docs/administrator-guide/overview.md]: Thanks - I've removed it.

File: docs/administrator-guide/incident-response-slash-commands.md [hunk 4]
```diff
@@ -1 +1,31 @@
...
+- `/incident info` - Show a summary of the current incident.
+- `/incident stage [next/prev]` - Move to the next or previous stage.
+
+## Adding slash commands to tasks
+
+Slash commands can be added to tasks to initiate actions as part of the incident response. 
+
+Here are some examples:
+
+- Add a task called **Sync up** with the slash command `/jitsi hello`. Running that slash command initiates a Jitsi call in the incident channel.
```
Comment (lieut-data) [inline docs/administrator-guide/incident-response-slash-commands.md:25]: Should we emphasize that the Jitsi plugin needs to be installed for this to work?

**Review by lieut-data — CHANGES_REQUESTED** [review]
Can we move the `assets/*` into `docs/assets/*`? I believe the `assets` folder is special in that these actually get bundled into the plugin for use by the server as needed.

Other than that, I just gave the remaining text a high level pass, and happy to just keep iterating here.

File: docs/administrator-guide/incident-response-slash-commands.md (hunk 4 above)
Comment (justinegeffen) [reply docs/administrator-guide/incident-response-slash-commands.md:25]: Good point. Yes.

Comment (justinegeffen) [inline docs/administrator-guide/incident-response-slash-commands.md:25]:
```suggestion
- Add a task called **Sync up** with the slash command `/zoom hello`. Running that slash command initiates a Zoom call in the incident channel.
```

**Review by lieut-data — APPROVED** [review]

**Pull request merged**